front/app/components/AvatarBubbles/index.tsx

```diff
@@ -128,96 +128,84 @@ interface DataProps {
 
 interface Props extends InputProps, DataProps {}
 
-interface State {}
-
 const defaultLimit = 4;
 
-class AvatarBubbles extends PureComponent<Props & InjectedIntlProps, State> {
-  static defaultProps = {
-    limit: defaultLimit,
-    size: 34,
-  };
-
-  render() {
-    const {
-      avatars,
-      avatarIds,
-      context,
-      size,
-      overlap,
-      userCount,
-      className,
-      intl: { formatMessage },
-    } = this.props;
-
-    if (!isNilOrError(avatars) && isNumber(userCount) && userCount > 0) {
-      const bubbleSize = (size as number) + 4;
-      const bubbleOverlap = overlap || 10;
-      const imageSize = bubbleSize > 160 ? 'large' : 'medium';
-      const avatarsWithImage = avatars.filter(
-        (avatar) =>
-          !isError(avatar) &&
-          avatar.attributes.avatar &&
-          avatar.attributes.avatar[imageSize]
-      ) as IAvatarData[];
-      const avatarImagesCount = avatarsWithImage.length;
-      const userCountBgColor =
-        this.props.userCountBgColor || colors.clIconSecondary;
-      const remainingUsers = userCount - avatarImagesCount;
-      const remainingUsersDigits = remainingUsers.toString().length;
-      const bubblesCount = avatarImagesCount + (remainingUsers > 0 ? 1 : 0);
-      const containerHeight = bubbleSize + 2;
-      const containerWidth =
-        bubblesCount * (bubbleSize - bubbleOverlap) + bubbleOverlap + 2;
-
-      if (avatarIds || context || avatarImagesCount > 0) {
-        return (
-          <Container
-            className={className}
-            width={containerWidth}
-            height={containerHeight}
-          >
-            {avatarsWithImage.map((avatar, index) => (
-              <AvatarImageBubble
-                key={index}
-                index={index}
-                overlap={bubbleOverlap}
+const AvatarBubbles = ({
+  avatarIds,
+  context,
+  size = 34,
+  overlap,
+  className,
+  userCountBgColor = colors.label,
```

Comment:
This is the only change relevant for a11y. By default, we have a darker background color for the final bubble that displays number of remaining accounts not shown in the avatar bubbles.